source/api/presentation/3.0/index.md

```diff
@@ -722,16 +722,17 @@ The value _MUST_ be an array of JSON objects. Each item _MUST_ have the `id` and
 
 #### 3.4.2. Internal Links
 
-##### within
-A link to another resource that contains this resource, such as a Manifest that is part of a Collection. When encountering the `within` property and the referenced resource is not included in the current representation, clients might retrieve the referenced resource to contribute to the processing of this resource. For example, if a Canvas is encountered as a stand alone resource, it might be `within` a Manifest that includes other contextual information, or a Manifest might be `within` a Collection, which would aid in navigation.
+##### partOf
+
+A link to another resource that contains this resource, such as a Manifest that is part of a Collection. When encountering the `partOf` property and the referenced resource is not included in the current representation, clients might retrieve the referenced resource to contribute to the processing of this resource. For example, if a Canvas is encountered as a stand alone resource, it might be part of a Manifest that includes other contextual information, or a Manifest might be part of a Collection, which would aid in navigation.
 
 The value _MUST_ be an array of JSON objects. Each item _MUST_ have the `id` and `type` properties, and _SHOULD_ have the `label` property.
 
- * Any resource types _MAY_ have the `within` property with at least one item<br/>
-   Clients _MAY_ render `within` on other resource types.
+ * Any resource types _MAY_ have the `partOf` property with at least one item<br/>
+   Clients _MAY_ render `partOf` on other resource types.
```

Comment:
I don't think "other resources types" makes sense when "Any resource type" can have `partOf`. 

Should it be "Clients _MAY_ render `partOf` links."?

Reply:
Fixed. Bad cut/paste error.

Reply:
And I see you fixed the plural that I missed :+1: